configstack/module.go

```diff
@@ -83,6 +84,16 @@ func resolveTerraformModule(terragruntConfigPath string, terragruntOptions *opti
 		return nil, err
 	}
 
+	// Fix for https://github.com/gruntwork-io/terragrunt/issues/208
+	matches, err := filepath.Glob(filepath.Join(filepath.Dir(terragruntConfigPath), "*.tf"))
+	if err != nil {
+		return nil, err
+	}
+	if (terragruntConfig.Terraform == nil || terragruntConfig.Terraform.Source == "") && matches == nil {
+		terragruntOptions.Logger.Printf("This module does not have an associated terraform config, sipping...")
```

Comment:
s/sipping/skipping/. Also, include the module path in this output so we know which module is being skipped!